source/includes/_widget.md

```diff
@@ -68,4 +68,32 @@ While the default theme doesn't provide any style rules for these they are avail
 | backstitch-instagram-result |
 | backstitch-youtube-result |
 | backstitch-linkedin-result |
-| backstitch-news-result |
\ No newline at end of file
+| backstitch-news-result |
+
+## Single Result Widget
+
+You can also use our widget to display a single result on your blog or website, as well. In order to do this, you must know the reference ID of the result you'd like to show, as well as the semantic type of the result (photo, article, status, video, service, product, or hotel). 
```

Comment:
backstitch also provides a widget to display a single result on your blog or website.